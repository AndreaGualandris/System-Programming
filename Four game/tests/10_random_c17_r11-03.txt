14
14
0
2
9
6
3
3
13
7
11
3
1
16
2
16
8
0
6
0
12
13
7
10
14
2
3
8
4
8
9
10
11
6
5
1
9
1
6
15
1
12
4
2
15
10
11
1
12
4
2
13
13
0
13
6
0
10
15
7
6
1
13
16
8
1
4
3
8
5
9
14
13
15
10
10
2
6
9
6
5
5
15
4
14
11
3
2
12
6
2
14
3
13
5
8
13
2
2
9
12
10
6
14
10
15
4
12
3
2
9
0
8
5
1
5
11
13
2
13
2
10
1
11
4
2
0
14
10
8
0
5
13
15
6
13
2
0
9
6
3
3
8
1
6
0
5
0
5
13
6
7
7
9
4
7
8
10
11
2
14
6
14
5
16
7
6
8
9
6
14
9
16
15
9
6
3
4
6
13
1
7
9
9
3
13
0
