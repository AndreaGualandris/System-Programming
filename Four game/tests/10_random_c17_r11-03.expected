-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
              o  
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
              x  
              o  
-----------------
01234567890123456
Player 'o' to move:
o
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
              x  
o             o  
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
              x  
o x           o  
-----------------
01234567890123456
Player 'o' to move:
         o
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
              x  
o x      o    o  
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
              x  
o x   x  o    o  
-----------------
01234567890123456
Player 'o' to move:
   o
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
              x  
o xo  x  o    o  
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
   x          x  
o xo  x  o    o  
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
   x          x  
o xo  x  o   oo  
-----------------
01234567890123456
Player 'x' to move:
       x
       v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
   x          x  
o xo  xx o   oo  
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
   x          x  
o xo  xx o o oo  
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
   x          x  
o xo  xx o o oo  
-----------------
01234567890123456
Player 'o' to move:
 o
 v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
   x          x  
ooxo  xx o o oo  
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
   x          x  
ooxo  xx o o oo x
-----------------
01234567890123456
Player 'o' to move:
  o
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
  ox          x  
ooxo  xx o o oo x
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
  ox          x x
ooxo  xx o o oo x
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
  ox          x x
ooxo  xxoo o oo x
-----------------
01234567890123456
Player 'x' to move:
x
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
x ox          x x
ooxo  xxoo o oo x
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
   x             
x ox  o       x x
ooxo  xxoo o oo x
-----------------
01234567890123456
Player 'x' to move:
x
v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
x  x             
x ox  o       x x
ooxo  xxoo o oo x
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
x  x             
x ox  o       x x
ooxo  xxoo oooo x
-----------------
01234567890123456
Player 'o' wins!
